src/main/java/core/basesyntax/operation/BalanceOperation.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.operation;
```

Comment:
The class `BalanceOperation` should not start with an empty line. According to the checklist, empty lines at the beginning of class definitions should be removed.